### PR TITLE
Fully switch to gradle dependency management

### PR DESCRIPTION
**Pull request by tokuhirom**

### CODE REVIEW

File: line-bot-model/src/test/java/com/linecorp/bot/model/event/ReplyEventTest.java [hunk 1]
```diff
@@ -16,15 +16,14 @@
 
 package com.linecorp.bot.model.event;
 
-import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.reflections.scanners.Scanners.SubTypes;
```
Comment (tokuhirom) [inline line-bot-model/src/test/java/com/linecorp/bot/model/event/ReplyEventTest.java:20]: remove spring dependencies from line-bot-model

File: line-bot-model/src/test/java/com/linecorp/bot/model/response/demographics/GetFriendsDemographicsResponseTest.java [hunk 2]
```diff
@@ -18,10 +18,10 @@
 
 import static java.lang.ClassLoader.getSystemResourceAsStream;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.springframework.util.StreamUtils.copyToByteArray;
 
 import java.io.IOException;
 
+import org.apache.commons.io.IOUtils;
```
Comment (tokuhirom) [inline line-bot-model/src/test/java/com/linecorp/bot/model/response/demographics/GetFriendsDemographicsResponseTest.java:24]: remove spring dependencies from line-bot-model

File: line-bot-parser/src/test/java/com/linecorp/bot/parser/LineSignatureValidatorTest.java [hunk 3]
```diff
@@ -19,9 +19,9 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.nio.charset.StandardCharsets;
+import java.util.Base64;
 
 import org.junit.jupiter.api.Test;
-import org.springframework.util.Base64Utils;
```
Comment (tokuhirom) [inline line-bot-parser/src/test/java/com/linecorp/bot/parser/LineSignatureValidatorTest.java:24]: remove spring dependencies from line-bot-parser

File: line-bot-spring-boot/build.gradle.kts [hunk 4]
```diff
@@ -15,6 +15,9 @@
  */
 
 dependencies {
+    implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+    annotationProcessor(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+
```
Comment (tokuhirom) [inline line-bot-spring-boot/build.gradle.kts:20]: line-bot-spring-boot should depends on spring-boot

**Review by dayflower — APPROVED** [review]
LGTM

**Pull request merged**